Chips/Chips.jsx

```diff
@@ -7,7 +7,7 @@ class ChipSet extends MaterialComponent {
   constructor() {
     super();
     this.componentName = 'chip-set';
-    this._mdcProps = ['choice', 'filter'];
+    this._mdcProps = ['choice', 'filter', 'input'];
```

Comment:
What about the [pre-selected](https://github.com/material-components/material-components-web/tree/master/packages/mdc-chips#pre-selected) chip?

Reply:
done